### ISSUE TITLE
Intermittent JSON Parsing Error

### ISSUE THREAD

**Issue by MaxMan79** [TheCraigHewitt/Seriously-Simple-Podcasting#177]
This issue has recently manifested itself, and I don't know enough to debug it properly.  My plugin version is 1.16.4, and I'm pretty sure I didn't see this issue until after I updated.  I'm using the [podcast_playlist] shortcode, and sometimes I will get the following JSON parse error, which causes the player to not appear:

**Uncaught SyntaxError: Unexpected token r in JSON at position 302
    at JSON.parse (<anonymous>)
    at Function.n.parseJSON (jquery.js:4)
    at Function.a.parseJSON (jquery-migrate.min.js:2)
    at n.initialize (wp-playlist.min.js:1)
    at n.e.View (backbone.min.js:1)
    at new n (backbone.min.js:1)
    at HTMLDivElement.<anonymous> (wp-playlist.min.js:1)
    at Function.each (jquery.js:2)
    at a.fn.init.each (jquery.js:2)
    at HTMLDocument.<anonymous> (wp-playlist.min.js:1)**

Refreshing the page usually resolves the issue and the player appears.  But sometimes it takes two or three refreshes.  I did a var_dump() to look at the JSON in question prior to it being parsed, and it was not malformed (the dump is shown below).  Comparing the JSON when the parse error occurs to the JSON when there is no parse error revealed that the JSON is exactly the same in both instances.  Any help would be greatly appreciated!

{"type":"audio","tracklist":true,"tracknumbers":true,"images":true,"artists":false,"tracks":[{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/475\/our-most-necessary-repentance.mp3","type":"audio\/mpeg","caption":"Michael Dirrim","title":"Our Most Necessary Repentance","description":"Jeremiah 6:17-24","meta":[],"image":"","thumb":""},{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/476\/is-anybody-listening.mp3","type":"audio\/mpeg","caption":"Michael Dirrim","title":"Is Anybody Listening?","description":"Jeremiah 6:24-31","meta":[],"image":"","thumb":""},{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/477\/a-pilgrims-progress.mp3","type":"audio\/mpeg","caption":"Ralph Bullard","title":"A Pilgrim's Progress","description":"Thanksgiving","meta":[],"image":"","thumb":""},{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/518\/another-hello.mp3","type":"audio\/mpeg","caption":"Ken Smith","title":"Another Hello","description":"Mark 1:10","meta":[],"image":"","thumb":""},{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/531\/another-one.mp3","type":"audio\/mpeg","caption":"Ralph Bullard","title":"Another One","description":"Matthew 1:1","meta":[],"image":"","thumb":""},{"src":"http:\/\/mydev.oklahomamtc.com\/podcast-player\/536\/the-next-one.mp3","type":"audio\/mpeg","caption":"Michael Dirrim","title":"The Next One","description":"Genesis 1:1","meta":[],"image":"","thumb":""}]}

**Comment by MaxMan79** [comment]
Finally figured this out, and it's not a SSP issue.  Like most Wordpress problems, it was caused by another plugin called RefTagger.  The other plugin added a script that was dynamically creating links for certain key words on the page, and this was injecting some HTML into the JSON after my var_dump(), which resulted in the parse error.  In case someone else runs into this problem, I fixed it by adding "script" to the list of tags that RefTagger ignores.  So this issue can be closed.  :)
